SettingsList.py

```diff
@@ -2575,6 +2587,25 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         shared         = True,
         disabled_default = 0,
     ),
+    Checkbutton(
+        name           = 'unlock_ganons_first_boss_door',
+        gui_text       = "Unlock Ganon's Tower First Boss Door",
+        default        = False,
+        gui_tooltip    = '''\
+            Unlocks the first boss key door in Ganon's Tower
+            to give access to the pots in the room.
+
+            The door leading to the upper stairway to
+            Ganondorf will still be locked.
+        ''',
+        gui_params     = {
+            'randomize_key': 'randomize_settings',
+            "hide_when_disabled": True
+        },
+        shared         = True,
+        disabled_default = False
+        
+    ),
```

Comment:
I question the necessity of this as a setting... I'd say either just always do it when pots are shuffled or don't do it.

Reply:
Maybe only do it with Triforce hunt? Would be strange to change vanilla item requirements here without it, but afaik the setting was added to make these pots not completely unreachable in TH.

Reply:
I thought the logic was fixed for that so it just wouldn't place TPs in there on TH, something with weirdness the way GBK is considered in TH

Reply:
Fenhl is correct that the logic was fixed so these pots shouldn't contain anything normally in TH. The setting itself was added as an option to make these pots less useless under certain settings (triforce hunt being one of them). I didn't want to just change the vanilla behavior with it being an explicit setting because I think that would have confused people too much.

Reply:
So how exactly do we come to a resolution on this? Some folks like it, some don't. I think it's a neat option for a particular few settings (triforce hunt, lower bridge requirements, etc.). The way it is now, it is not enabled by default so it shouldn't take anyone by surprise. I can work on the wording of the setting description as r0bd0g recommended.

Reply:
I can see the arguments for wanting it either way, so I guess keeping it as a setting is fine